code/controllers/subsystem.dm

```diff
@@ -78,7 +78,7 @@
 	can_fire = 0
 	flags |= SS_NO_FIRE
 	Master.subsystems -= src
-	return ..()
+	. = ..()
```

Comment:
??

Reply:
???
It's functionally the same, just more consistent.

Reply:
I wasn't sure if you were making a stylistic change or if you wanted them to return a different GC hint but didn't include the full change. I'm not sure what GC hint is appropriate here.

Reply:
The PR I am stealing from returns `QDEL_HARRDEL_NOW`. Was going to ask @Lohikar but Discord is crapping out at the moment.